demos/codeluppi-2018/Snakefile

```diff
@@ -0,0 +1,43 @@
+include: "../common.smk"
+configfile: "config.yml"
+
+BASE_URL = "https://s3.amazonaws.com/vitessce-data/0.0.31/master_release/linnarsson/linnarsson"
```

Comment:
Ok, I get this is to save time, but why do we need to save time?  Isn't working with "raw data" better?  I think this is too far along to change now, but just curious.

Reply:
Yes I agree it would be better. It was more to save my own time developing the data processing scripts, and also because the raw files for this dataset are pretty large.